packages/spacecat-shared-gpt-client/README.md

```diff
@@ -68,6 +71,38 @@ async function fetchInsights(prompt) {
 fetchInsights('How can we improve customer satisfaction?');
 ```
 
+#### Via Chat Completions endpoint
+
+```javascript
+async function fetchInsights(prompt) {
+  try {
+    const client = FirefallClient.createFrom({
+      env: {
+        FIREFALL_API_ENDPOINT: 'https://api.firefall.example.com',
+        FIREFALL_API_KEY: 'yourApiKey',
+        FIREFALL_API_CAPABILITY_NAME: 'yourCapabilityName',
+        IMS_HOST: 'ims.example.com',
+        IMS_CLIENT_ID: 'yourClientId',
+        IMS_CLIENT_CODE: 'yourClientCode',
+        IMS_CLIENT_SECRET: 'yourClientSecret',
+      },
+      log: console,
+    });
+    const options = {
+      imageUrls: ['data:image/png;base64,iVBORw0KGgoAAAA...=']
+    };
+
+    // // Internally, use Firefall's .../chat/completions endpoint.
```

Comment:
Like above I think this can be added as a short description above the code block. Also, Firefall is an implementation detail that users of the gpt client don't really need to know about. There are a bunch of `FIREFALL_*` env options however so I guess users of this client do need to know that Firefall is being used.